### DISCUSSION TITLE
Letest version

### DISCUSSION THREAD

**Discussion by amitmiglani91** [strimzi/strimzi-kafka-operator#7841] [Q&A]
https://github.com/strimzi/strimzi-kafka-operator/blob/main/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

this we can use in deployment right?

**Comment by scholzj** [comment]
Well, you will need the other YAMLs from that folder as well. But yes, that is the latest release 0.32. You can check the different installation methods and the guides for them in the docs if you want: https://strimzi.io/docs/operators/latest/full/deploying.html#con-strimzi-installation-methods_str

**Reply by amitmiglani91** [reply]
Thanx a lot yha for topic and Producer & consumer i needed, just for installation or config setting this is enough. thanx for replying , highly appreciated.
